governance/pyth_staking_sdk/src/pyth-staking-client.ts

```diff
@@ -274,30 +274,26 @@ export class PythStakingClient {
     return sendTransaction([instruction], this.connection, this.wallet);
   }
 
-  public async stakeToPublisher(options: {
```

Comment:
@keyvankhademi FYI I updated the api signatures for a few functions to make them less verbose, I don't think it's less clear though but PTAL and let me know if you're OK with these changes

Reply:
In a typed language like strict ts or rust the type of a variable also brings information when working with the code. I think the `address` in `stakeAccountPositionsAddress: PublicKey` is redundant because the type PublicKey already shows that this is an address. Having said that I'm not necessarily opposed to this change. Feel free to keep it this way.

Reply:
Yeah, I agree with you, but in this case it felt weird to name it `stakeAccountPositions` because my expectation from that variable name is that it's a list of positions.

Would `stakeAccount` be a more correct name?